### ISSUE TITLE
RightCurly: False negative on multiblock tokens with ALONE_OR_SINGLELINE option

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#4091]
```
$ cat TestClass.java
public class TestClass {
    void method() {
        if (true) {
            getClass();}
    }
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
        <module name="RightCurly">
            <property name="option" value="ALONE_OR_SINGLELINE"/>
        </module>
    </module>
</module>

$ java -jar checkstyle-7.6-all.jar -c TestConfig.xml TestClass.java
Starting audit...
Audit done.
```
Expected: violation on `IF`
Issue is reproducible with other multiblock tokens as well (for, while...)

**Comment by Vladlis** [comment]
I am on it

**Comment by rnveach** [comment]
Fix is merged
